meetings/2023-01-04.md

```diff
@@ -29,36 +39,63 @@
+* SC: To date, the index document only listed documents published under `TR/`.
+* SC: The notifications-panel also has https://github.com/solid/notifications/blob/main/README.md#subscription-types that lists all types.
+* SC: Question: Would it be simpler to re-use https://solidproject.org/TR/ as the index for auxiliary documents (e.g., notification channel types), since it is intended to list all of the CG's Work Items? The "Indexing Procedure" can be part of TRs that need it.
+* SC: Quick Overview: a document that would hold things that need to be registered and are typically dependent on other specifications. Notifications protocol allows auxiliary specs that define individual notification channel types, for example.  Similar to [DID method approach](https://www.w3.org/TR/did-spec-registries/#did-methods).
+* We have published web socket subscription type (with corresponding document under TR). Terminology has been changed, from *Subscription types* to *channel types*. It would be incorrect to rely on notification subscription types. If this approach is continued a new document set is needed.
+* SC: Instead of recreating documents or reusing, we could use the solid/TR, regardless of maturity. The TR document would be the central document with links to all related documents. 
+* PC: Suggest W3C doesn't require a separate document, but this should be verified. Propose using the TR document as a registry of SOLID and related specifications. Registries could be considered different (i.e., entries of a given registry are homogeneous). A registry is defined by a process. The TR Index document list has a heterogeneous mix of documents. We could use that document to hold all the registries that are need, but it would be ideal if it structured.
+* SC: We had an extensive talk regarding the Notifications Panel and a decision was to use the separate document. If we change names again, the TR space may become confusing. IS there quick fix for the TR? Any question or comments?
+* SC: Would a section in the TR dedicated to registration channel types suffice?
+* PC: yes, that is my proposal.
+* SC: A decision now is not needed, but we can take this approach if the panel (the only users to date), we can use it. If there are other specs in the future we can take that route as well.
+* SC: There are not other comments, we can move to the next topic.
+
+
 ### Add TR/2022/notifications-protocol-20221231
 https://github.com/solid/specification/pull/491
 
+* SC: Notifications Panel will create a new release. It is versioned at 0.2. Current version in TR is v0.1. The decision regarding numbering has breaking changes from v0.1. Since it is not a v1.0 release and there are not enough implementations and test suites, we held back on labelling version 1.0. We don't expect major changes moving forward (sans editorial changes). 
```

Comment:
```suggestion
* SC: Notifications Panel will create a new release, versioned at 0.2. Current version in TR is v0.1. This version has breaking changes from v0.1. Since it was not a v1.0 release and there are not enough implementations and test suites, we held back on labeling as version 1.0. We don't expect major changes moving forward (sans editorial changes). 
```